docs(continuous_replication): rename to incremental_replication

thomas-dkmt opened the pull request:
Since continuous replication was renamed to "incremental replication," the [Continuous Replication](https://docs.xen-orchestra.com/continuous_replication) page in the Xen Orchestra documentation had already been updated to reflect this change. However, the page URL still retained the "continuous_replication" terminology.

Meanwhile, there was a [duplicate page](https://docs.xen-orchestra.com/incremental_replication) identical to the continuous_replication page, but with the correct URL ("incremental_replication"). This duplicate page was never included in the navigation bar.

To ensure consistency with our terminology and to avoid having duplicate or unreferenced pages, this PR deletes the continuous_replication page and has the incremental_replication page properly linked in the navigation bar.

Additionally, a mention of "CR" on the page has been rewritten as "incremental replication" to reduce ambiguity.

thomas-dkmt committed:
doc(XO): Update URL of the Incremental replication page

Continuous replication was renamed as incremental replication.
The relevant page was already renamed but not the URL.
Also, a duplicate page with the correct URL already existed but was not linked in the navigation bar.
The Continuous replication page is now deleted and
the Incremental replication page is now linked in the nav bar.

Signed-off-by: Thomas Moraine <thomas.moraine@vates.tech>

thomas-dkmt committed:
doc(XO): Clarify CR initials to avoid ambiguity

Previously, "CR" was used as an abbreviation for "Continuous Replication." However, this was found to be ambiguous and unclear.
Since "Continuous Replication" has been renamed to "Incremental Replication,"
we have updated the documentation to use the full term "incremental replication" instead.

Signed-off-by: Thomas Moraine <thomas.moraine@vates.tech>

nathanael-h reviewed (changes requested):
This PR is good, :+1: 
May I suggest to add a redirect from `/continuous_replication` to `/incremental_replication`, using https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-client-redirects so that the old links people might have in bookmarks, forum topics and support tickets is still valid?

olivierlambert commented:
Good idea. Also not the right place but I wonder if we planned to change it in XO 5 UI or just wait for XO 6?

thomas-dkmt committed:
docs(XO) Redirect obsolete URL

The continuous_replication URL doesn't exist
anymore and is replaced with incremental_replication.
A redirection has been added to reflect that

Signed-off-by: Thomas Moraine <thomas.moraine@vates.tech>

nathanael-h commented:
Question for @lsouai-vates I think. Indeed in XO5 we still use "Continuous" 
![image](https://github.com/user-attachments/assets/e1e1ebf0-694c-4f38-b8ea-bb27e869504e)

lsouai-vates commented:
it's a good question, I am asking XO team about it

lsouai-vates commented:
@nathanael-h and @thomas-dkmt I can confirm that we will use incremental instead of continuous
